src/stack-hci/setup.py

```diff
@@ -12,7 +12,7 @@
 # HISTORY.rst entry.
 VERSION = '0.1.4'
```

Comment:
Please increase the version to 0.1.5; otherwise this change cannot be released as 0.1.4 has already released.

Reply:
I see, my mistakes, I borrowed such version from "HISTORY.rst" since previous highest version is 0.1.3.... Anyway, I will update it then.

Reply:
Please add history note for 0.1.4 as well

Reply:
Added.